hazelcast/src/main/java/com/hazelcast/instance/GroupProperties.java

```diff
@@ -203,7 +203,31 @@
     public static final String PROP_ELASTIC_MEMORY_UNSAFE_ENABLED = "hazelcast.elastic.memory.unsafe.enabled";
     public static final String PROP_ENTERPRISE_LICENSE_KEY = "hazelcast.enterprise.license.key";
     public static final String PROP_MAP_WRITE_BEHIND_QUEUE_CAPACITY = "hazelcast.map.write.behind.queue.capacity";
+
+    /**
+     * Defines event queue size for WAN replication, only valid for Hazelcast Enterprise
```

Comment:
What does event queue size mean apart from the size of the event queue? Please provide insight in how fits into the picture and what is being influenced if the property is changed.